espressif/idf-eclipse-plugin
# IEP-923 add esp32c6 support

## Description

Added a new supported toolchain, launch target, and cmake file related to esp32c6. Requires latest esp-idf with esp32c6 support

Fixes # ([IEP-923](https://jira.espressif.com:8443/browse/IEP-923))

## Type of change

Please delete options that are not relevant.
- New feature (non-breaking change which adds functionality)


## How has this been tested?

Test 1:
- install tools with esp-idf which supports the esp32c6 target.
- verify that the target bar is updated with the esp32c6 target
- go to preferences -> Toolchains -> verify that esp32c6 toolchain is added
- go to preferences -> cmake -> verify that esp32c6 toolchain file is added 
Test 2:
- create a new project, and build with an esp32c6 target. Verify that the project has been built according to the target
Test 3:
- create a debug configuration
-  edit debug configuration, select esp32c6 target -> verify that gdb executable, config options, SVD path are correct

**Test Configuration**:
* ESP-IDF Version:
* OS (Windows,Linux and macOS):

## Dependent components impacted by this PR:
- Build
- Debug
- Toolchain list
- CMake toolchain files list

## Checklist
